app/models/migration/cohort.rb

```diff
@@ -1,4 +1,7 @@
 module Migration
   class Cohort < Migration::Base
+    def next
```

Comment:
I wonder if its worth adding tests for the migration models if we're adding in logic 🤔 I think we did in NPQ reg towards the end. It can be one for another ticket though.